.devcontainer/devcontainer.json

```diff
@@ -12,9 +12,14 @@
 		"--cap-add=SYS_PTRACE",
 		"--security-opt=seccomp:unconfined",
 		"--security-opt=apparmor:unconfined",
-		"--volume=/tmp/.X11-unix:/tmp/.X11-unix"
+		"--volume=/tmp/.X11-unix:/tmp/.X11-unix",
+		"--gpus" "all",
 	],
-	"containerEnv": { "DISPLAY": "${localEnv:DISPLAY}" },
+	// default
+	//"containerEnv": { "DISPLAY": "${localEnv:DISPLAY}"}
+	// enable instead the two lines below for nvidia driver + qt apps 
+	"containerEnv": { "DISPLAY": "${localEnv:DISPLAY}" , 
+				"QT_X11_NO_MITSHM": "1"},
```

Comment:
Is there something special about setting this variable here vs in the dockerfile?

Reply:
it seems not, I moved it!